polynote-frontend/polynote/ui/input/plot_selector.ts

```diff
@@ -1109,10 +1109,9 @@ export class PlotSelector extends Disposable {
                 ]),
                 hasDimensions ? this.facetCheckbox = checkbox(['facet'], 'Facet', !!(state.facet)).onValueChange<boolean>(checked => {
                     if (checked) {
-                        this.el.classList.add('facet');
                         this.stateHandler.updateField("facet", s => s === undefined ? setValue({}) : NoUpdate )
                     } else {
-                        this.el.classList.remove('facet');
+                        this.stateHandler.updateField("facet", s => s !== undefined ? setValue(undefined) : NoUpdate )
```

Comment:
This is actually an unrelated bugfix - I noticed that unchecking "facet" didn't do anything.